aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1640,7 +1641,28 @@ def swagger_handler():
                 license_info=license_info,
             )
 
-            body = generate_swagger_html(spec, swagger_js, swagger_css)
+            # The .replace('</', '<\\/') part is necessary to prevent a potential issue where the JSON string contains
+            # </script> or similar tags. Escaping the forward slash in </ as <\/ ensures that the JSON does not
+            # inadvertently close the script tag, and the JSON remains a valid string within the JavaScript code.
+            escaped_spec = model_json(
+                spec,
+                by_alias=True,
+                exclude_none=True,
+                indent=2,
+            ).replace("</", "<\\/")
```

Comment:
Hello @rubenfonseca! I moved this logic to this file because we can use the same specification in JSON View and SwaggerUI rendering.
Please let me know if make sense.